tests/nnstreamer_filter_tensorflow_lite/runTest.sh

```diff
@@ -96,7 +96,7 @@ function run_pipeline() {
 }
 
 arch=$(uname -m)
-if [ "$arch" = "aarch64" ] || [ "$arch" = "armv7l" ]; then
+if [ "$arch" = "armv7l" ]; then
```

Comment:
Why aarch64 is disabled?

Reply:
This was wrong change. Has been reverted.